src/app/frontend/create/from/file/component.ts

```diff
@@ -43,12 +43,17 @@ export class CreateFromFileComponent extends ICanDeactivate {
     return !this.file || this.file.content.length === 0 || this.create_.isDeployDisabled();
   }
 
-  create(): void {
+  async create(): Promise<void> {
```

Comment:
I am actually not a big fan of using async/await and `Promise<void>` without returning anything from the function. Here it's used more as a workaround or alternative code rather than for any good reason. Using `then`, `catch`, and `finally` is shorter and easier to read for me.

Reply:
Yeah this makes sense. Thanks for the suggestion.